### PR TITLE
Issue #5891: renamed package grammars to grammar

### PR DESCRIPTION
**Pull request by mismatch**
Issue #5891 
Renamed package grammars to grammar

### CODE REVIEW

**Comment by mismatch** [comment]
Can someone explain me what's wrong with my PR?

File: pom.xml [hunk 1]
```diff
@@ -1041,7 +1041,7 @@
         <version>2.2</version>
         <configuration>
           <sourceDirectory>
-            ${basedir}/src/main/resources/com/puppycrawl/tools/checkstyle/grammars
+            src/main/resources/com/puppycrawl/tools/checkstyle/grammar
```
Comment (romani) [inline pom.xml]: Please restore basedir usage, if it if not required, it should be done in separate commit or PR.

**Comment by romani** [comment]
@mismatch , please address https://travis-ci.org/checkstyle/checkstyle/jobs/406904364#L657 , just add reference to issue in PR description

**Comment by mismatch** [comment]
Thank you, @romani
I've returned ${basedir} in pom.xml and added PR description.
It seems that the only blocker is spellchecker for now.

**Comment by romani** [comment]
@mismatch,

spellchecker's issue is fixed in master, to pass all CIs, please rebase on latest master code.

**Comment by mismatch** [comment]
I've rebased my branch on master but still no luck :(

https://travis-ci.org/checkstyle/checkstyle/jobs/407145407 failed because of timeout. Command is "./.ci/travis/travis.sh test-de"
Any thoughts?

**Comment by romani** [comment]
Travis build item was restarted.

**Comment by mismatch** [comment]
Finally, everything is green. I look forward to accepting my PR.

**Comment by romani** [comment]
@mismatch , thanks a lot for contribution, let us know if you have time to contribute more, we can suggest next issue, or you can choose your self.

**Comment by mismatch** [comment]
@romani, I'm going to spend some hours to contribute on the weekend but cannot promise for sure.

**Comment by romani** [comment]
Non of us have any obligation or promises, we just do smth when we have time.
